### ISSUE TITLE
[Feature-Request] Send notification to specific topic of group with topics

### ISSUE THREAD

**Issue by LasseRosenow** [filipnet/checkmk-telegram-notify#11]
Is it somehow possible to send a notification to a specific topic of a telegram group, that has enabled the new "topic" feature?

See: https://telegram.org/blog/topics-in-groups-collectible-usernames

**Comment by filipnet** [comment]
Since only groups with more than 200 members can activate topics, this feature request can unfortunately not be tested.
This is not a bug, so this issue will be closed. Please check the API specifications from Telegram.

**Comment by LasseRosenow** [comment]
But this information is outdated.

They changed it and now even small groups can exist. 

I myself just created one only having 5 members :)

Please reopen

**Comment by filipnet** [comment]
That is correct, I reopen the topic as a "Feature Request". However, I myself unfortunately don't have the capacity to implement this request in the code. Maybe this information will help in the short term:
https://codex.so/telegram-bots-in-group-topics

**Comment by eworm-de** [comment]
I could have a look... Any preference on how to pass the thread id? As a fourth parameter?

**Comment by filipnet** [comment]
Sure, that would definitely enhance the solution. I find the `Custom User Attributes` very interesting (see README.md). In the long run, I would like to move away from parameters entirely and only specify the `Telegram API token` in the `Notification Rule`.

All other parameters should be handled as `Custom Attributes` in the future, like this:

![Image](https://github.com/user-attachments/assets/d35ff737-0661-4495-abe0-8cd67e4d8fd7)

One thing to keep in mind: Using a `fourth parameter` might cause issues if people choose **not** to use the Privacy Mode. In that case, the order of parameters could become problematic. What do you think about it?

**Comment by eworm-de** [comment]
> Sure, that would definitely enhance the solution. I find the `Custom User Attributes` very interesting (see README.md). In the long run, I would like to move away from parameters entirely and only specify the `Telegram API token` in the `Notification Rule`.

Sounds good, but let's not do that here.

> One thing to keep in mind: Using a `fourth parameter` might cause issues if people choose **not** to use the Privacy Mode. In that case, the order of parameters could become problematic. What do you think about it?

Exactly, that's why I asked. See my pull request... I append this to the group id in second parameter.

@LasseRosenow, want to give it a try?
